app/models/shipit/pull_request.rb

```diff
@@ -102,7 +102,7 @@ def required_statuses
     end
 
     def self.schedule_merges
-      Shipit::Stack.where(id: pending.uniq.pluck(:stack_id)).find_each(&:schedule_merges)
+      Shipit::Stack.where(merge_queue_enabled: true).find_each(&:schedule_merges)
```

Comment:
Note: That query won't be indexed, but the table isn't very likely to be big enough to matter.